feature/search/src/main/java/com/terning/feature/search/search/component/ImageSlider.kt

```diff
@@ -15,16 +14,20 @@ import androidx.compose.runtime.remember
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.layout.ContentScale
-import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.unit.dp
+import coil3.compose.AsyncImage
+import coil3.request.ImageRequest
+import coil3.request.crossfade
 import com.terning.core.designsystem.extension.noRippleClickable
 import com.terning.core.designsystem.theme.Grey200
+import com.terning.domain.search.entity.SearchBanner
 import kotlinx.coroutines.delay
 
 @Composable
 fun ImageSlider(
     modifier: Modifier = Modifier,
-    images: List<com.terning.domain.search.entity.SearchBanner>,
+    searchBanners: List<SearchBanner>,
     onAdvertisementClick: (Int) -> Unit,
```

Comment:
저희 List는 ImmutableList로 수정해주면 감사하겠습니다🙇‍♀️
```suggestion
    searchBanners: ImmutableList<SearchBanner>,
    onAdvertisementClick: (Int) -> Unit,
```